cves/CVE-2016-3092.yml

```diff
@@ -145,12 +142,17 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
   commits:
-    - commit:
-      note:
-    - commit:
-      note:
+    - commit: bcefe58374c421c9431c144938e2e91d22f5061b
+      note: |
+        This commit was interesting because it made some major changes to how the
```

Comment:
Maybe a good solution would have been to not append input directly in the first place?

Reply:
This is an interesting thought. I will address this in my edit.